# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../workbone/qd-fixes.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./hardware.c
+++ ./hardware.c	Mon Oct 18 16:16:27 1999
@@ -14,6 +14,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <sys/types.h>
+#include <linux/types.h>
 #include <sys/ioctl.h>
 #include <fcntl.h>
 #include <sys/param.h>
@@ -25,9 +25,6 @@
 #    else
 #     include <sundev/srreg.h>
 #    endif /* linux */
-#ifdef __GLIBC__
-#include <sys/ustat.h>
-#endif
 #include "struct.h"
 #include "workbone.h"
